test/inductor/test_torchinductor_opinfo.py

```diff
@@ -220,7 +220,6 @@ def format_op(op):
     "nn.functional.avg_pool1d": {i64},
     "nn.functional.avg_pool2d": {i64},
     "nn.functional.local_response_norm": {i64},
-    "nn.functional.rrelu": {f32, f64},
```

Comment:
I think these will still fail because of rng issues. we can fix that in another pr. One fix is to not assert equal outputs of tensors if rng was used, another would be to use `torch/_inductor/config.fallback_random = True`.

Reply:
@eellison I left this in because it's what exposed the problem with the schema you described. Lemme know if you prefer to handle it in a different PR tho.

Reply:
Nice, looks good! i meant a more general mechanism to avoid `assert_equal` when rng is present or enable fallback_random. This is a good one off for this test!